lib/module/xkcd.go

```diff
@@ -0,0 +1,88 @@
+package module
+
+import (
+	"fmt"
+	"github.com/davidscholberg/irkbot/lib/configure"
+	"github.com/davidscholberg/irkbot/lib/message"
+	"github.com/nishanths/go-xkcd"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const apiUrlFmtDefine = "https://relevantxkcd.appspot.com/process?%s"
+
+func Helpxkcd() []string {
+	s := "xkcd <search> - find a xkcd comic relevant to the search term"
+	return []string{s}
+}
+
+//Perform the actual GET and return the resulting body as a string
+func get(apiURL string) (string, error) {
+	response, err := http.Get(apiURL)
+	if err != nil {
+		return "", err
+	}
+	defer response.Body.Close()
+	bodyBytes, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return "", err
+	}
+	bodyString := string(bodyBytes)
+	return bodyString, nil
+}
+
+//Parse the body string for the comic number we want
+func parseString(bodyString string) string {
```

Comment:
Instead of a single return value that may or may not be an error message, this function should return a string and an `error` object, just like the `get` function. You can create an error object with the [fmt.Errorf](https://golang.org/pkg/fmt/#Errorf) function.

Further, the error object should contain a descriptive error message, not just "error", which doesn't explain anything.